### PR TITLE
Create Riemann Health Monitor alert plugin

### PR DESCRIPTION
**Pull request by sharms**

### CODE REVIEW

File: bosh-monitor/lib/bosh/monitor/plugins/riemann.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        logger.info("Riemann delivery agent is running...")
+        return true
+      end
+
+      def validate_options
+       !!(options.kind_of?(Hash) && options["host"] && options["port"])
+      end
+
+      def client
+        @client |= Riemann::Client.new host: options["host"], port: options["port"]
```
Comment (cppforlife) [inline bosh-monitor/lib/bosh/monitor/plugins/riemann.rb]: single `|`?

**Comment by sharms** [comment]
Good catch - that was a typo, the pull has been updated

**Comment by sharms** [comment]
We also don't understand the CI Gemfile issue - we added riemann-client to the Gemspec for monitoring

**Comment by jmcarp** [comment]
Sorry about the failing builds earlier, but tests are passing now, and this should be ready for review.

**Comment by dpb587-pivotal** [comment]
Looks good to me for a squash+merge. Independently verified the new, vendored gem checksums as of c62a51cba3d022e9520a42af76bfde982ada0e3c.

**Comment by dpb587-pivotal** [comment]
Merge note - make it to `develop` instead of `master`.

**Pull request closed without merging**